partitionmanager/cli.py

```diff
@@ -360,7 +360,10 @@ def do_partition(conf):
         except partitionmanager.types.DatabaseCommandException as e:
             log.warning("Failed to automatically handle %s: %s", table, e)
             metrics.add("alter_errors", table.name, 1)
-        except partitionmanager.types.TableEmptyException:
+        except (
+            partitionmanager.types.TableEmptyException,
+            partitionmanager.types.NoValidRatesOfChangeException,
+        ):
             log.warning("Table %s appears to be empty. Skipping.", table)
```

Comment:
should we log which of the two exceptions we got?

Reply:
Good idea.